docs/zkapps/how-zkapps-work.mdx

```diff
@@ -98,11 +101,11 @@ When you deploy to a new Mina address, the Mina Protocol will charge a 1 MINA fe
 <img src="/img/7_zkApps_DeploySmartContract.jpg" width="95%" />
 <br></br>
 
-Developers usually build a UI to allow users to interact with the smart contract.
+zkApp developers usually build a UI to allow users to interact with the smart contract.
 
-Typically, this is a static website deployed to a host of the developer’s choosing. We recommend web hosts that offer a global CDN to ensure the best user experience.
+Typically, this static website is deployed to a host of the developer’s choosing.  Recommend web hosts offer a global CDN to ensure the best user experience.
```

Comment:
`Typically, this` > `This`

I think the 2nd sentence is now unclear because there is no actor and `that` is needed here. I'd keep the original. I tried rewriting it to achieve your goal: `Web hosts that offer a global CDN provide the best user experience for your zkApp` This feels a bit disconnected from the proceeding thought, but if feel strongly against `we` it could work.

Reply:
another teaching moment. (yes, sorry, that r/w was clunky)

Let's see how this r/w sounds?